src/dbnode/storage/database_test.go

```diff
@@ -1032,6 +1033,104 @@ func TestWideQuery(t *testing.T) {
 	assert.Equal(t, exSpans, spanStrs)
 }
 
+func TestReadMismatches(t *testing.T) {
+	ctrl := xtest.NewController(t)
+	defer ctrl.Finish()
+
+	d, mapCh, _ := defaultTestDatabase(t, ctrl, BootstrapNotStarted)
+	defer func() {
+		close(mapCh)
+	}()
+
+	commitlog := d.commitLog
```

Comment:
```suggestion
	commitLog := d.commitLog
```
Will solve collision with imported package name (also consistency).
Same in `TestWideQuery`.